pyscript.core/tests/integration/test_01_basic.py

```diff
@@ -191,7 +203,7 @@ def test_packages(self):
             "hello asciitree",  # printed by us
         ]
 
-    @pytest.mark.skip("FIXME: No banner")
+    @pytest.mark.skip("NEXT: No banner")
```

Comment:
if by *banner* we mean the DOM related to the error, it's a `div.py-error` from the *error* plugin.